packages/demo/src/assets/versions.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "title": "v5.3.1",
+    "version": "5.3.1",
+    "description": "Pattern documentation, code snippets and implementation guidelines for the Design System Styles.",
+    "url": "https://6401b58ebf0f4e298ee2dbe7--swisspost-web-frontend.netlify.app/",
+    "dependencies": {
+      "@angular/core": "14.2.12",
+      "@ng-bootstrap/ng-bootstrap": "12.1.2",
+      "@swisspost/design-system-intranet-header": "3.1.1",
+      "@swisspost/design-system-styles": "5.4.1",
+      "bootstrap": "5.1.3"
```

Comment:
The versions are [miss matched](https://ng-bootstrap.github.io/#/getting-started#dependencies). Do you think we could do something about that?

Reply:
Yes, we just re-write the versions.json.

Reply:
ng-bootstrap 12 is theoretically not compatible with Angular > 14.1
Support was added as part of [13.0.0](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CHANGELOG.md#1300-2022-07-28) (the same version that removed ngb buttons)

We should probably downgrade to Angular 14.0.0 or do you think that's not a problem?

Reply:
As discussed, we'll keep it like this because our tests with this version combo were successful and because a newer Version with compatible Angular/ng-bootstrap versions is available.